code/package.json

```diff
@@ -72,7 +72,7 @@
     "@vscode/iconv-lite-umd": "0.7.0",
     "@vscode/policy-watcher": "^1.1.4",
     "@vscode/proxy-agent": "^0.18.1",
-    "@vscode/ripgrep": "^1.15.6",
+    "@vscode/ripgrep": "^1.15.7",
```

Comment:
Also we should get https://github.com/microsoft/vscode-ripgrep/pull/54 merged and THEN bump this to 1.15.8

Reply:
ping @Sachin-Itagi , https://github.com/microsoft/vscode-ripgrep/pull/54 is merged now.

Reply:
the version is going to be updated in the upstream soon https://github.com/microsoft/vscode/pull/198880/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R75